security/pkg/monitoring/monitoring.go

```diff
@@ -0,0 +1,147 @@
+// Copyright 2019 Istio Authors
```

Comment:
I like the sharing of this, but maybe we should break this into a common place in this PR so we don't have the duplication. @douglas-reid any thoughts?

Reply:
@howardjohn I agree that this should be moved over. But that's probably too much to take on in this PR.  What do you think about going forward with this PR and filing an issue to track the adding of a monitoring package to `istio.io/pkg` ?